src/EditorFeatures/Core/Options/CompletionOptions.cs

```diff
@@ -10,5 +10,10 @@ internal static class EditorCompletionOptions
 
         [ExportOption]
         public static readonly Option<bool> UseSuggestionMode = new Option<bool>(FeatureName, nameof(UseSuggestionMode), defaultValue: false);
+
+        // Default into suggestion mode in the watch/immediate windows but respect the
+        // user's preferences if they switch away from it.
+        [ExportOption]
+        public static readonly Option<bool> UseSuggestionMode_Debugger = new Option<bool>(FeatureName, nameof(UseSuggestionMode_Debugger), defaultValue: true);
```

Comment:
The intent is this isn't persisted somewhere? i.e. next VS run this is back to what it was before?

(Not applying judgement, just confirming intent.)